namespaces/mson-namespace.md

```diff
@@ -75,8 +91,14 @@ Note: In MSON DOM _Nested Member Types_ _Type Section_ is the `content` of the e
             - sample
             - default
     - `variable` (boolean) - Element content is _Variable Value_
-    - `sample` (*T*) - Alternative sample value for _Member Types_
-    - `default` (*T*) - Default value for _Member Types_
+    - `sample` - Alternative sample value for _Member Types_
+
+		The type of of `sample` attribute MUST match the type of element's `content`.
+
+    - `default` - Default value for _Member Types_
+
+		The type of of `default` attribute MUST match the type of element's `content`.
+
```

Comment:
This is just a thought, but could this concept ever be useful in MSON? That is, do you ever see a need to be able to express the type of one value must match the type of another, where that other value may be an enum?

Just a thought. Not an issue here.